### ISSUE TITLE
ABS migration

### ISSUE THREAD

**Issue by gianfranco-l** [giantswarm/roadmap#1607]

**Comment by uvegla** [comment]
Mentioned this on @giantswarm/sig-releng today.

**Comment by uvegla** [comment]
Happa migration to ABS: https://github.com/giantswarm/happa/pull/3861

**Comment by uvegla** [comment]
> Happa migration to ABS: https://github.com/giantswarm/happa/pull/3861 + related change in architect-orb: https://github.com/giantswarm/architect-orb/pull/399

We need to support having multiple charts in the same repository to some extent because the Happa use case makes sense / would be hard to migrate fe-docs to a separate repository because the contents are generated based on code / tests for Happa.

**Comment by weatherhog** [comment]
@uvegla as you have been working on this in 2022. Can you tell from your last comment what is missing to proceed?

**Comment by uvegla** [comment]
I think the multi chart support needs to be checked (and who knows what else, so needs some research + picking up context). That is potentially mostly for edge scenarios. Then a guide could be written on how to upgrade (potentially migrate some to get some experience) and then I would say teams need to migrate their apps. Or HB does it for everyone, which is a (lengthy?) migration process as usual.

**Comment by weatherhog** [comment]
@uvegla what do you think. Is this still needed?

**Comment by uvegla** [comment]
Yes, but there is another one that is more recent and has more info / more people know about it: https://github.com/giantswarm/giantswarm/issues/31101.

Closing this in favour of that.
